tree/tree/inc/TTree.h

```diff
@@ -160,6 +163,11 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    TTree(const TTree& tt);              // not implemented
    TTree& operator=(const TTree& tt);   // not implemented
 
+#ifdef R__USE_IMT
+   mutable Bool_t fIMTFlush{false};               ///<! True if we are doing a multithreaded flush.
```

Comment:
Odd to see a mutable non-atomic.  Is that really the intention? (i.e. is it really never possible to modify/read it in parallel?)